lib/hashes.md

```diff
@@ -5,15 +5,16 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+Hashes are sort of like dictionaries. It is a collection of keys and values where you can refer to a value using the associated key, or vice versa.
```

Comment:
Can you experiment with some Ruby code on Repl.it and write some more about the **vice-versa** assertion? Hashes don't work quite the same for keys as they do for values, so I'd like you to provide some code and commentary to illustrate your understanding.